### ISSUE TITLE
nltk.draw.pos-concordance(), mentioned on website, does not exist

### ISSUE THREAD

**Issue by alanfgh** [nltk/nltk#4]
From website ( http://nltk.googlecode.com/svn/trunk/doc/howto/corpus.html#tagged-corpora ):

"Use nltk.draw.pos-concordance() to access a GUI for searching tagged corpora."

However, the command fails with "AttributeError: 'module' object has no attribute 'pos'".

Similar problem if I replace the hyphen with an underscore.

Yet concordance.py does exist:

http://code.google.com/p/nltk/source/browse/trunk/nltk/nltk/draw/concordance.py?spec=svn8391&r=7460

though the path at the top of the page is confusing, due to the double "nltk":

"svn/  trunk/ nltk/ nltk/ draw/ concordance.py"

The command nltk.draw.demo() did work, and according to the source code, it called pos_concordance:

def demo():
    pos_concordance()

But all the demo seemed to do was allow one to move graphical elements around in the window, flip a tree from vertical to horizontal arrangement, and so on. I had been hoping it would allow one to drill down on a component or otherwise perform a function other than cosmetic rearrangement. At the least, I would think it would cast light on how the four trees were created via the API.

To summarize:
(1) The website refers to what seems to be a nonworking call.
(2) When I look in the source code and try to exercise the closest thing to that call, I get less than I was expecting.

**Comment by stevenbird** [comment]
These HOWTO documents are in the process of being migrated to github.  The problem is fixed in the doctest source file, and will be reflected in the online pages as soon as possible.
